charts/argo-cd/README.md

```diff
@@ -105,6 +105,9 @@ For full list of changes please check ArtifactHub [changelog].
 
 Highlighted versions provide information about additional steps that should be performed by user when upgrading to newer version.
 
+### 5.36.0
+This version adds `global.env` for all deployed deployments
```

Comment:
This PR does not require specific action (e.g. breaking change) to users, so I think we don't need this note. 🙋

Reply:
deleted